app/templates/events/event_list.html

```diff
@@ -30,23 +30,22 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
     </div>
   </div>
 </div>
-<div>
+<div class="row">
   <ul class="nav nav-tabs nav-fill mx-3 mb-3" id="pills-tab" role="tablist">
-  <li class="nav-item" role="presentation">
+  <li class="col-lg-3 col-md-6 col-sm-12 col-xs-12 nav-item" role="presentation">
```

Comment:
We don't need to specify all of these, since we only need to break at a certain screensize, and they should all be equal width. It should work to give each of them `col-sm`, `col-md`, or `col-lg`, depending on which one of those causes the break at the right place.